packages/cactus-plugin-ledger-connector-quorum/Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM cactus-api-server:latest
+
+ARG NPM_PKG_VERSION=latest
+
+RUN npm i @hyperledger/cactus-plugin-keychain-memory@${NPM_PKG_VERSION} --production
```

Comment:
@jordigiam Sorry, missed this one earlier: The keychain memory plugin => only good for development but this image is meant for production so I was wondering if it's possible/doable/easy enough to remove the keychain plugin from the image itself and then have it install at runtime (by adding an entry for it in the `PLUGINS` array in the config of the examples so that the examples still work, but the keychain memory plugin is not hardcoded in the image)
Apologies once again for missing this earlier!

Reply:
Done!